docs/notebooks/beta_regression.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>Maybe the following is a little simpler?</p><p> </p><pre class="ql-syntax" spellcheck="false"># Mean Values
alpha, beta = mukappa_to_alphabeta(
    expit(fitted.posterior["Intercept"].values.mean()),
    fitted.posterior["probabilities_kappa"].values.mean() 
)

# Upper / lower 95%
hdi = az.hdi(fitted.posterior, hdi_prob=.95)

intercept_hdi = expit(hdi["Intercept"].values)
kappa_hdi = hdi["probabilities_kappa"].values

alpha_lower, beta_lower = mukappa_to_alphabeta(
    intercept_hdi[0],
    kappa_hdi[0]
)
alpha_upper, beta_upper = mukappa_to_alphabeta(
    intercept_hdi[1],
    kappa_hdi[1]
)
              
print(f"Alpha - mean: {np.round(alpha)}, 95% HDI: {np.round(alpha_lower)} - {np.round(alpha_upper)}")
print(f"Beta - mean: {np.round(beta)}, 95% HDI: {np.round(beta_lower)} - {np.round(beta_upper)}")
</pre><p> </p><p>Feel free to change it or adapt. The only thing I would like to avoid are the <code>eval().item()</code>  calls</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/442/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>